lib/imagefs.coffee

```diff
@@ -280,3 +327,13 @@ exports.listDirectory = (definition) ->
 			listDirectory(disk, definition.partition, definition.path)
 	else if definition.image instanceof filedisk.Disk
 		listDirectory(definition.image, definition.partition, definition.path)
+
+###*
+# @summary Closes the allocated resources. Call this when you're done using imagefs if you expect the program to end.
+# @function
+# @public
+#
+# @returns {Promise}
+###
+exports.close = ->
+	ext2fs.close()
```

Comment:
The node-ext2fs creates a mutex when it is imported.
See https://github.com/resin-io/node-ext2fs/blob/master/src/async.cc#L22 .
It prevents node from exiting so you need to explicitly close the resources when you're done using the library: https://github.com/resin-io/node-ext2fs/blob/master/src/async.cc#L28 .
In the case of a server you don't have to worry about this but in a CLI for example, you'll have to call this at some point.

fatfs doesn't have this issue.

I don't get the problem with having the `ext2fs` a module global.
Would it be better if I prevented close from being called twice?